.github/workflows/generate-allure-report.yml

```diff
@@ -188,6 +188,18 @@ jobs:
           base-path: "/home/runner/work/catalyst-voices/catalyst-voices/"
           parallel: true
 
+      - name: Coveralls
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: coverallsapp/github-action@v2
+        with:
+          path-to-lcov: "${{ env.COVERAGE_REPORT_PATH }}/api-tests.coverage.info"
+          allow-empty: true
+          debug: true
+          flag-name: api-test
+          base-path: "/home/runner/work/catalyst-voices/catalyst-voices/"
```

Comment:
What about coverage in /catalyst_voices_packages? Should we collect it too?

Reply:
Yes you are right! I just added it